### PR TITLE
fix backslashes for path additional files in args

### PR DESCRIPTION
**Pull request by burnedoutman**

### CODE REVIEW

**Comment by Flowseal** [comment]
What's the purpose of backslashes in the args?

**Comment by ghost** [comment]
> What's the purpose of backslashes in the args?

Presumably at some stage of running the BAT file `set ARGS= ...` will be wrapped in quotes and the service will not be created without backslashes

File: service_discord.bat [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
---filter-udp=50000-65535 --dpi-desync=fake,tamper --dpi-desync-any-protocol --dpi-desync-fake-quic="%~dp0quic_initial_www_google_com.bin" --new ^
---filter-tcp=443 --hostlist="%~dp0list-discord.txt" --dpi-desync=fake,split2 --dpi-desync-autottl=2 --dpi-desync-fooling=md5sig --dpi-desync-fake-tls="%~dp0tls_clienthello_www_google_com.bin"
+--filter-udp=443 --hostlist=\"%~dp0list-discord.txt\" --dpi-desync=fake --dpi-desync-udplen-increment=10 --dpi-desync-repeats=6 --dpi-desync-udplen-pattern=0xDEADBEEF --dpi-desync-fake-quic=\"%~dp0quic_initial_www_google_com.bin\" --new ^
+--filter-udp=50000-65535 --dpi-desync=fake,tamper --dpi-desync-any-protocol --dpi-desync-fake-quic=\"%~dp0quic_initial_www_google_com.bin\" --new ^
+--filter-tcp=443 --hostlist=\"%~dp0list-discord.txt\" --dpi-desync=fake,split2 --dpi-desync-autottl=2 --dpi-desync-fooling=md5sig --dpi-desync-fake-tls=\"%~dp0tls_clienthello_www_google_com.bin\"
 
-set SRVCNAME=zapret
+set SRVCNAME=zapret_for_discord
```
[the remark this answers is not in the document]
Comment (burnedoutman) [reply service_discord.bat:7]: fixed

**Comment by Flowseal** [comment]
@burnedoutman This pull request is not about separating **discord** and **discord_youtube** services (and that's not needed, because running both services may cause undefined behavior). Let's stop only on backslashes, I agree with you about them.

**Comment by burnedoutman** [comment]
@Flowseal ok, done

**Comment by Flowseal** [comment]
LGTM

**Review by Flowseal — APPROVED** [review]

**Pull request merged**